plasma_framework/python_tests/tests/contracts/root_chain/test_start_in_flight_exit.py

```diff
@@ -95,19 +92,19 @@ def test_start_in_flight_exit_with_erc20_token_and_eth_should_succeed(testlang,
 
     # Inputs are correctly set
     input_info = in_flight_exit.get_input(0)
-    assert input_info.owner == to_canonical_address(owner.address)
-    assert input_info.token == NULL_ADDRESS
+    assert input_info.exit_target == owner.address
+    assert input_info.token == NULL_ADDRESS_HEX
```

Comment:
Might consider to have a const `ETH` or `ETH_TOKEN` with the value of `NULL_ADDRESS_HEX` to make this more readable.